### ISSUE TITLE
Automatic error report

### ISSUE THREAD

**Issue by dominicwllmsn** [spyder-ide/spyder#7648]
### What steps will reproduce the problem?

Trying to drag and drop a .py file into the tab bar in spyder

### Traceback
```python-traceback
  File "/anaconda3/lib/python3.6/site-packages/spyder/widgets/editor.py", line 2389, in dropEvent
    files = [f for f in files if encoding.is_text_file(f)]
TypeError: 'NoneType' object is not iterable
```

## Package Versions

* Spyder version: 3.2.8 
* Python version: 3.6.5
* Qt version: 5.9.4
* PyQt5 version: 5.9.2
* Operating system: Darwin 17.7.0

### Dependencies

```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.4.0 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.23.0 (OK)
numpy >=1.7      :  1.14.3 (OK)
sphinx >=0.6.6   :  1.7.4 (OK)
rope >=0.9.4     :  0.10.7 (OK)
jedi >=0.9.0     :  0.12.0 (OK)
psutil >=0.3     :  5.4.5 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.28.2 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.4.0 (OK)
pylint >=0.25    :  1.8.4 (OK)
```

**Comment by ccordoba12** [comment]
I think this is fixed in our latest version (3.3.0). Please update by opening Terminal.app and running there

    conda update spyder

and try again.

**Comment by dominicwllmsn** [comment]
Thanks for you rhlep.
